Update cloud_cost_management.md (#14435)
DataDog/documentation: content/en/infrastructure/cloud_cost_management.md

### Configure the AWS integration

Select your AWS management account from the dropdown. Once your account is selected, Datadog displays tags associated with that account. If you have multiple similarly-named management accounts, viewing the tags associated with a selected account can help ensure that you've selected the specific one you want.

> **Note**: we recommend that you send us a Cost and Usage Report from an [AWS _management account_][6] for cost visibility into related _member accounts_. If you decide to send us a Cost and Usage report from an AWS _member account_, please ensure that you've selected following options in [preferences][7] of the _management account_ so that we have full visibility into the member account:

Reviewer: ```suggestion
**Note**: Datadog recommends that you send a Cost and Usage Report from an [AWS **management account**][6] for cost visibility into related **member accounts**. If you decide to send a Cost and Usage report from an AWS **member account**, ensure that you've selected the following options in [preferences][7] of the **management account** so that Datadog has full visibility into the member account:
```

> <img width="770" alt="Screen Shot 2022-06-27 at 1 53 10 PM" src="https://user-images.githubusercontent.com/99217719/176004906-d06096a2-cec8-481a-8113-0a98bc963d9a.png">

Reviewer: Several issues with this image:
- All images should be uploaded to the static/images folder
- Using an image makes the content inaccessible to people using screen readers.

Please remove this image and instead list the options as text.

### Configure the AWS integration

> Select your AWS management account from the dropdown. Once your account is selected, Datadog displays tags associated with that account. If you have multiple similarly-named management accounts, viewing the tags associated with a selected account can help ensure that you've selected the specific one you want.

Reviewer 2: ```suggestion
Select your AWS management account from the dropdown menu, allowing Datadog to display tags associated with this account. If you have multiple similarly-named management accounts, view the tags associated with a selected account to ensure you have selected the specific account you want.
```

> **Note**: Datadog recommends that you send a Cost and Usage Report from an [AWS **management account**][6] for cost visibility into related **member accounts**. If you decide to send a Cost and Usage report from an AWS **member account**, ensure that you've selected the following options in [preferences][7] of the **management account** so that Datadog has full visibility into the member account:

Reviewer 2: ```suggestion
**Note**: Datadog recommends sending a Cost and Usage Report from an [AWS **management account**][6] for cost visibility into related **member accounts**. If you decide to send a Cost and Usage report from an AWS **member account**, ensure that you have selected the following options in your **management account's** [preferences][7] to allow Datadog to have full visibility into the member account:
```

> * Linked Account Access
> * Linked Account Refunds and Credits
> * Linked Account Discounts

Reviewer 2: ```suggestion
* **Linked Account Access**
* **Linked Account Refunds and Credits**
* **Linked Account Discounts**
```
